data/tlg0064/tlg001/tlg0064.tlg001.1st1K-grc1.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
-    <teiHeader>
+    <teiHeader xml:lang="eng">
         <fileDesc>
             <titleStmt>
                 <title xml:lang="lat">Periplus Hannonis</title>
                 <author>Hanno</author>
-                <editor xml:lang="lat">CAROLUS MULLERUS</editor>
-                <sponsor>HARVARD COLLEGE LIBRARY</sponsor>
+                <editor xml:lang="lat">Carolus Müllerus</editor>
+                <sponsor>Harvard College Library</sponsor>
```

Comment:
Replace from the `<sponsor>` element to the last `</respStmt>` with the following:

```
<funder>Harvard Library Arcadia Fund</funder>
<principal>Gregory Crane</principal>
<respStmt>
    <persName xml:id="DDD">Digital Divide Data</persName>
    <resp>Corrected and encoded the text</resp>
</respStmt>
<respStmt>
    <persName>Gregory Crane</persName>
    <resp>Editor-in-Chief, Perseus Digital Library</resp>
</respStmt>
<respStmt>
    <persName>Matt Munson</persName>
    <resp>Project Manager (University of Leipzig), 2016 - present</resp>
</respStmt>
<respStmt>
    <persName>Annette Gessner</persName>
    <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
</respStmt>
<respStmt>
    <persName>Thibault Clérice</persName>
    <resp>Lead Developer (University of Leipzig) 2015 - 2017</resp>
</respStmt>
<respStmt>
    <persName>Bruce Robertson</persName>
    <resp>Technical Advisor</resp>
</respStmt>
```

Reply:
Generally the diagrams in this collection appear in the Praefatio or in the notes, which we didn't OCR (because they're in Latin). Anyways, if there was a diagram, I'd like to point to a scanned page with the illustration, or to the specific section of that page (e.g. with a facs attribute).